changed navbar to be more consistent with other THON sites

vishnkr opened the pull request.

vishnkr committed:
changed navbar to be more consistent with other THON sites

adonato98 reviewed (changes requested):
This is a really good start. I have thought of a couple changes to the color palette as well as some other random stuff:

- [ ] Instead of "Log in with your Penn State credentials" on one line, keep "Log in" on the heading line, and add "Use your Penn State credentials" as a subheading.

- [ ] Add the word "Mailtrain" back in to the right of the new THON logo so it reads "THON Mailtrain." User your best judgement on either gold like the language dropdown or white.

- [ ] Crop out "thon.org" from the logo so we are just seeing the shuttershade

- [ ] Change the secondary navbar / footer background color to #f5f5f5 b/c it is in our branding already

- [ ] Change the footer text to reflect that we are running a fork of Mailtrain, not Mailtrain itself. You can have it read "Fork of Mailtrain v2, modified by Penn State Dance Marathon under a GNU Public License v3.0. © 2018 Mailtrain.org, info@mailtrain.org. Source on GitHub" (with hyperlinks from before)

- [ ] Change references to "MailTrain" to "Mailtrain" on the landing page. This one's on me, my bad :)

- [ ] Look into changing the primary button color from the lighter blue to more of a #3C4CA3 (pulled from THINK's secondary colors). Hopefully there is one file that controls all the button colors, if there isn't then we can talk about leaving this till later.

vishnkr committed:
design update

- added mailtrain to navbar (need to fix breadcrumb taking up navbar space)
- changed color of nav-items
- modified footer text
- shuttershade logo fix

vishnkr committed:
added more PR changes

- footer color changed to #f5f5f5
- changed button colors
- fixed breadcrumbs error

jasonchhay reviewed (commented):
<img width="376" alt="image" src="https://user-images.githubusercontent.com/28989541/85495372-61503100-b5a8-11ea-8366-11a349d04640.png">

Looks good with me, only thing is I'm thinking maybe change the breadcrumbs to a lighter hue of the navy blue, or revert it to the original hyperlink color. That way it'll help it stand out that these are meant to be clickable.

vishnkr committed:
revert breadcrumbs color

jasonchhay committed:
Updated favicon, removed extra image files

jasonchhay committed:
Readded app-footer class
